### PR TITLE
Add option "Make payer optional"

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

File: includes/class-wc-gateway-swedbank-pay-checkout.php [hunk 1]
```diff
@@ -178,22 +190,24 @@ public function __construct() {
...
+		$this->auto_capture           = isset( $this->settings['auto_capture'] ) ? $this->settings['auto_capture'] : $this->auto_capture;
+		$this->logo_url               = isset( $this->settings['logo_url'] ) ? $this->settings['logo_url'] : $this->logo_url;
+		$this->instant_checkout       = isset( $this->settings['instant_checkout'] ) ? $this->settings['instant_checkout'] : $this->instant_checkout;
+		$this->checkin                = isset( $this->settings['checkin'] ) ? $this->settings['checkin'] : $this->checkin;
+		$this->checkin_country        = isset( $this->settings['checkin_country'] ) ? $this->settings['checkin_country'] : $this->checkin_country;
+		$this->terms_url              = isset( $this->settings['terms_url'] ) ? $this->settings['terms_url'] : get_site_url();
+		$this->save_cc                = 'no';
+		$this->use_payer_info         = isset( $this->settings['use_payer_info'] ) ? $this->settings['use_payer_info'] : $this->use_payer_info;
+		$this->use_cardholder_info    = isset( $this->settings['use_cardholder_info'] ) ? $this->settings['use_cardholder_info'] : $this->use_cardholder_info;
```
Comment (asbjornu) [inline includes/class-wc-gateway-swedbank-pay-checkout.php]: The `cardHolder` information is always optional and should only be sent if it is available. It doesn't make sense requiring this information, I think.

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: includes/class-wc-gateway-swedbank-pay-checkout.php [hunk 2]
```diff
@@ -129,6 +129,18 @@ class WC_Gateway_Swedbank_Pay_Checkout extends WC_Payment_Gateway {
...
+	 * Send payer info
+	 * @var string
+	 */
+	public $use_payer_info = 'yes';
+
+	/**
+	 * Send cardholder info
+	 * @var string
+	 */
+	public $use_cardholder_info = 'yes';
```
Comment (asbjornu) [inline includes/class-wc-gateway-swedbank-pay-checkout.php]: Does setting `$use_cardholder_info` to  `'yes'` mean that it is required and must always be sent? It should not be required and should only be sent if it is made available by the WooCommerce installation somehow.

**Review by asbjornu — COMMENTED** [review]
I don't understand what is going to happen when this is merged. Will `cardholder` data still be sent if it is available? Can you please describe in the PR exactly what the intention of the code is and what is supposed to change when this is merged?

**Comment by asbjornu** [comment]
@aait, @olegisk, why was this closed? Please write your intentions behind pull requests you submit and when they are closed.

**Comment by aait** [comment]
> @aait, @olegisk, why was this closed? Please write your intentions behind pull requests you submit and when they are closed.

I've close this PR because we got rid of "Send cardholder info" feature, so there is nothing to commit.

**Comment by asbjornu** [comment]
> I've close this PR because we got rid of "Send cardholder info" feature, so there is nothing to commit.

So now we aren't sending `cardholder` information at all? Or only when available?

**Pull request closed without merging**